src/controllers/action-queue/runtime.js

```diff
@@ -54,31 +54,27 @@ export const executeActionProgram = async (account: EdgeAccount, program: Action
   }
 
   // Await Effect
-  let checkErrors: Error[] = []
+  let retryDelay = 2000
   while (true) {
     if (effect == null) break
 
     try {
       const { isEffective, delay } = await checkActionEffect(account, effect)
 
-      // Reset error aggregation (and failure count)
-      checkErrors = []
-
       // Break out of effect check loop if the ActionEffect passes the check
       if (isEffective) break
 
       // Delay next check
       await snooze(delay)
     } catch (err) {
-      checkErrors.push(err)
+      console.warn(`Action effect check failed:\n\t${String(err)}`)
+      console.error(err)
```

Comment:
console.warn for both?

Reply:
Warn would be good for both. I'd really like to only have one warn here, but I'm not sure how to add some context to the error (`"Action effect check failed:..."`) without mucking with the stack trace.

Reply:
Unfortunately, RN messed this one up for us; console.error is the only log util we have to maintain the original error stack trace. We'll need to figure out a way to manage our log levels and have traceability for these error types. But until then, it's more useful to us to have the stack trace.

I'm hoping that either we can get a patch for this in RN itself, or that the reason the stack traces are not there are by some other mutation to the `console` object by our codebase so we can fix it (hopeful guess).